hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -1453,6 +1453,15 @@ public Optional<CompactionContext> requestCompaction(int priority,
     // Before we do compaction, try to get rid of unneeded files to simplify things.
     removeUnneededFiles();
 
+    if (this.region.getTableDescriptor().isSkipMergingCompaction()) {
+      if (!conf.getBoolean("hbase.store.delete.expired.storefile", true)) {
+        LOG.info("Because the conf of hbase.store.delete.expired.storefile is false, so the table "
```

Comment:
Can we rephrase as "Since config 'hbase.store.delete.expired.storefile' is set to false, ignoring SKIP_MERGING_COMPACTION set at table level"